templates/dashboard/product/list.html

```diff
@@ -50,20 +50,7 @@
         <form method="POST" action="{% url 'dashboard:product-bulk-update' %}" novalidate id="bulk-actions-form">
           {% csrf_token %}
           <input name="action" type="hidden" id="bulk-action" />
-          <nav class="data-table-header-alternative single">
-            <div class="bulk-info show-bulk">
-              <div class="data-table-header-action-selected-items col inline"></div>
-            </div>
-            <div class="bulk-actions show-bulk nav-wrapper">
-              <ul class="right">
-                {% for action, label in bulk_action_form.action.field.widget.choices %}
-                  <li>
-                    <a href="" data-action="{{ action }}" class="btn-flat waves-effect">{{ label|upper }}</a>
-                  </li>
-                {% endfor %}
-              </ul>
-            </div>
-          </nav>
+          {% include 'dashboard/product/_bulk_actions_bar.html' %}
```

Comment:
Make sure that this template is versatile as possible, as it will be used also with other tables. For sure the file should be available in a more general directory, e.g. `dashboard/includes`.